src/components/photography/top-news.js

```diff
@@ -51,7 +52,7 @@ const CategoryContainer = styled.div`
 
 const Card = styled.div`
   position: absolute;
-  background-color: rgba(255, 255, 255, 0.9);
+  background-color: ${colorOpacity['white_0.9']};
   border: 2px solid black;
```

Comment:
Do we need to change `black` to `colorGrayscale.black`

Reply:
fixed ([d30b42](https://github.com/twreporter/twreporter-react/pull/2772/commits/d30b42cd3cca96f616a7bdaf6e20d94e6c523100))